tethysext/atcore/templates/atcore/map_view/map_view.html

```diff
@@ -20,25 +20,27 @@
     {% block map_tabs %}
       <ul class="nav nav-tabs atcore-nav-tabs" role="tablist">
         {# LAYERS TAB #}
-        <li role="presentation" {% block disable_default_tab %}class="active"{% endblock %}>
-          <a id="layers-tab-toggle"
+        <li class="nav-item" role="presentation">
+          <a class="nav-link {% block disable_default_tab %}active{% endblock %}"
+             id="layers-tab-toggle"
              href="#layers-tab-panel"
              aria-controls="layers-tab-panel"
              aria-selected="true"
              role="tab"
-             data-toggle="tab">{{ layer_tab_name }}</a>
+             data-bs-toggle="tab">{{ layer_tab_name }}</a>
         </li>
         {# LEGEND TAB #}
         {% if show_legends %}
           {% block legend_tab %}
             {% if legends %}
-            <li role="presentation" {% block disable_legend_tab %}{% endblock %}>
-              <a id="legend-tab-toggle"
+            <li class="nav-item" role="presentation" {% block disable_legend_tab %}{% endblock %}>
```

Comment:
Move `{% block disable_legend_tab %}` to the `<a>` tag as above. The block is poorly named, but is used for the same purpose.